azure-pipelines.yml

```diff
@@ -123,7 +123,7 @@ jobs:
       # Moving connection data to arguments
       # AZURE_STORAGE_CONNECTION_STRING: $(AZURE_STORAGE_CONNECTION_STRING)
       ACCOUNT_NAME: $(AZURE_STORAGE_ACCOUNT)
-      ACCOUNT_KEY: $(AZURE_STORAGE_ACCESS_KEY)
+      ACCOUNT_KEY: "$(AZURE_STORAGE_ACCESS_KEY)="
```

Comment:
Is the extra `=` here intentional?

Reply:
yes, I keep getting an authentication error, both with the connection string and the accounts name+key, so that's a test to see if the problem is that the `=` at the end of the base64 is missing (the error message says incorrect padding, so sounded like this could be the problem)